tests/test_gcc.py

```diff
@@ -0,0 +1,43 @@
+"""Tests for the gcc containers."""
+import pytest
+from pytest_container import container_and_marks_from_pytest_param
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerData
+
+from bci_tester.data import GCC_CONTAINERS
+
+CONTAINER_IMAGES = GCC_CONTAINERS
+
+_HELLO_VERSION = "2.12.1"
+
+CONTAINERFILE_HELLO = f"""
+WORKDIR /src
+RUN curl -sLO https://ftpmirror.gnu.org/hello/hello-{_HELLO_VERSION}.tar.gz && \\
+    tar --no-same-permissions --no-same-owner -xf hello-{_HELLO_VERSION}.tar.gz && \\
+    cd hello-{_HELLO_VERSION} && \\
+    ./configure && \\
+    make && \\
+    make install
+"""
+
+HELLO_CONTAINERS = [
+    pytest.param(
+        DerivedContainer(
+            base=container_and_marks_from_pytest_param(ctr)[0],
+            containerfile=CONTAINERFILE_HELLO,
+        ),
+        marks=ctr.marks,
+        id=ctr.id,
+    )
+    for ctr in CONTAINER_IMAGES
```

Comment:
`CONTAINER_IMAGES` is only necessary when you are using the `auto_container` fixture (that one parametrizes automatically).

```suggestion
_HELLO_VERSION = "2.12.1"

CONTAINERFILE_HELLO = f"""
WORKDIR /src
RUN curl -sLO https://ftpmirror.gnu.org/hello/hello-{_HELLO_VERSION}.tar.gz && \\
    tar --no-same-permissions --no-same-owner -xf hello-{_HELLO_VERSION}.tar.gz && \\
    cd hello-{_HELLO_VERSION} && \\
    ./configure && \\
    make && \\
    make install
"""

HELLO_CONTAINERS = [
    pytest.param(
        DerivedContainer(
            base=container_and_marks_from_pytest_param(ctr)[0],
            containerfile=CONTAINERFILE_HELLO,
        ),
        marks=ctr.marks,
        id=ctr.id,
    )
    for ctr in GCC_CONTAINERS
```

Reply:
@dcermak we could use auto_container going forward? I find it useful to have the infrastructure for that in place already.

Reply:
There's nothing stopping you from using `auto_container` right now

Reply:
it doesn't have a use here.